app/src/main/res/layout/activity_product_detail.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Views go off the screen when rotated

Reply:
Also, none of these views match constraints, can you change that rather than using match_parent or fixed dimensions?